apps/sports-helsinki/src/domain/venue/venueKeywords/__tests__/VenueKeywords.test.tsx

```diff
@@ -33,7 +33,9 @@ it('should render keywords and handle click', async () => {
     screen.getByRole('link', { name: new RegExp(keywordNames[0], 'i') })
   );
   expect(router).toMatchObject({
-    asPath: `/haku?q=${encodeURIComponent(capitalize(keywordNames[0]))}`,
+    asPath: `/haku?q=${encodeURIComponent(
+      capitalize(keywordNames[0])
+    )}&searchType=Venue`,
```

Comment:
Maybe would be safer to construct this test like the actual query, ie. with the `PARAM_SEARCH_TYPE` constant.

Reply:
Changed this to contruct the query same way as it does in VenueKeywords